docs/com.bitwarden.organizations.md

```diff
@@ -0,0 +1,79 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden organization
+
+The `com.bitwarden.organization` doctype is used to group
+[ciphers](./com.bitwarden.ciphers.md) in a single unit and/or to share them with
+other [bitwarden contacts](./com.bitwarden.contacts.md).
+
+## Attributes
+
+- `name`: {string} - the name of the organization
+- `members`: {object} - the members of the organization (see below)
+- `defaultCollection`: {object} - the collection of the organization (see below)
+- `cozyMetadata`: {object} - the [cozyMetadata](README.md#document-metadata)
+
+## Members
+
+An organization can be shared with a list of members.
+
+- `email`: {string} - the email of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `key`: {string} - the public key of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `name`: {string} - the name of the member
+- `owner`: {bool} - true if the member owns the organization
+- `status`: {int} - the status of the member (`0` = `invited`, `1` = `accepted`, `2` = `confirmed`)
+- `user_id`: {string} - the id of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+
+Note that the `name` and `key` attributes may be missing if the member status is still `invided` as those data are sent only when he accepts the first [Cozy sharing](./io.cozy.sharings.md) from organization's owner.
+
+## Collection
+
+A collection is a container for [ciphers](./com.bitwarden.ciphers.md).
+
+A [ciphers](./com.bitwarden.ciphers.md) can live alone or in a collection and a collection lives in an organization.
```

Comment:
Small reformulation proposition: `A [cipher](./com.bitwarden.ciphers.md) can live alone or included in a collection. Any collection is linked to an organization.`

Reply:
File edited ✔️